src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Square extends Figure {
```

Comment:
Don't begin class or method implementation with an empty line. The empty line after the class declaration can be removed.